apps/functions/githubWebhook/status.ts

```diff
@@ -0,0 +1,20 @@
+import {StatusEvent} from '@octokit/webhooks-types';
+import {firestore} from 'firebase-admin';
+// TODO(josephperrott): Remove usage of FirestoreReference and fromFirestoreReference.
```

Comment:
OOC: Why is this something you want to remove? Is that mainly a todo because you want to have better helpers?

Reply:
Yeah, I think we should be able to do this without having to have these helpers used this way, but I couldn't for the life of me figure it out at the time.  I was planning to just come back to it.